src/platforms/dotnet/profiling/index.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Set Up Profiling
+description: "Learn how to enable profiling in your app if it is not already set up."
+sidebar_order: 5000
+---
+
+<PlatformContent includePath="profiling/index/preface" />
+
+With [profiling](/product/profiling/), Sentry allows you to collect and analyze performance profiles from real user devices in production to give you a complete picture of how your application performs in a variety of environments.
+
+<Note>
+
+.NET profiling alpha is available for:
+- .NET 6.0 and above for Windows, Linux and macOS using [.NET Core CLR Diagnostics](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/diagnostics-client-library)
```

Comment:
This is good context, but better to keep it for some detailed docs further down below. Here it's just too much context

> using [.NET Core CLR Diagnostics](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/diagnostics-client-library)


```suggestion
- .NET 6.0 and above for Windows, Linux and macOS
```